torch/distributed/rpc/__init__.py

```diff
@@ -50,15 +49,24 @@ def init_rpc(
                         ``Worker1``) Name can only contain number, alphabet,
                         underscore, and/or dash, and must be shorter than
                         128 characters.
-            init_method(str): backend specific init arguments.
+            init_method(str): URL specifying how to initialize the
+                                     RPC backend. Default is "env://" if no
+                                     ``init_method`` is specified.
             rank (int): a globally unique id/rank of this node.
             world_size (int): The number of workers in the group.
             rpc_backend_options (RpcBackendOptions): The options passed to RpcAgent
```

Comment:
(This can come in a followup PR) Do we have docs for rpc_backend_options to show how to construct it, probably with a customized init_method?

Reply:
I'll work on this in a follow up PR.